src/app.ts

```diff
@@ -17,8 +20,13 @@ app.use(logger("dev"));
 app.use(express.static("public"));
 app.use(favicon(__dirname + "/public/favicon.ico"));
 
+
 // Use main router for API routes
 app.use("/api/v1", mainRouter);
 
 
-export default app;
\ No newline at end of file
+app.use(notFoundMiddleware); // 404 Not Found handler
+app.use(errorHandlerMiddleware); // Global error handler
```

Comment:
These comments are unnecessary, I understand from the name of the variables what these are doing.